docs/validators/guides/block-production.md

```diff
@@ -0,0 +1,84 @@
+# Block production
+Koinos uses the novel consensus algorithm, Proof-of-Burn (PoB). To participate in block production you will need to configure your node as well as perform some simple actions on the blockchain.
+
+---
+## Proof-of-Burn basics
+Proof-of-Burn has similarities to both Proof-of-Work (PoW) and Proof-of-Stake (PoS) with some notable differences. Like Proof-of-Stake, no mining occurs to produce blocks; this places all physical nodes on an equal playing field. Hashing is simulated in Proof-of-Burn by burning KOIN in order to gain Virtual Hash Power (VHP). Aside from using VHP rather than physical machines performing hashes, the algorithm works similarly to Proof-of-Work.
+
+---
+## Prerequisites
+Please follow our guide on [running a node](running-a-node.md). Be sure to start the node with the `--profile all` or `--profile block_producer` option to start the optional block producer microservice.
+
+---
+## Retrieving your address and key
+Upon starting a new node, a private block production key will be automatically generated for you. This key will be at `$KOINOS_BASEDIR/block_producer/private.key` (`$KOINOS_BASEDIR` is `~/.koinos` on macOS/Linux and `C:\koinos` for Windows if following our guide). There is a corresponding `public.key` file that is written out when the block producer runs and is always the public key corresponding to the private key that the block producer is configured to use (`private.key` by default).
+
+You will want to copy the contents of `public.key` and save this for later use. This is our "hot" key.
+
+```console
+cat $KOINOS_BASEDIR/block_producer/public.key
+```
+```{ .txt, .no-copy }
+Aq4Ps_Ch-f8OZDnpQOov2SiMvdYyA5tn0oWa36QWnTeH
+```
+
+Next, you will want your main account to hold VHP and KOIN for block production. Let us use the `koinos-cli` to register the PoB contract and open the wallet for our main account.
```

Comment:
Should we link to the CLI documentation in case this is their first encounter with the CLI?